### ISSUE TITLE
linking to figures in the article metadata in Zenodo

### ISSUE THREAD

**Issue by myrmoteras** [plazi/Biodiversity-Literature-Repository#52]
@gsautter can you remind us why to include the link to the PNG of an image and not a DOI as a the persistent identifier of it? this is clearly not an identifier?!

{"relation": "cites", "identifier": "https://zenodo.org/record/1458424/files/figure.png"}

**Comment by gsautter** [comment]
Agreed, that's surely not the intended identifier. I'll change the JSON accordingly.

The reason we hold on to the PNG URLs (as HTTP URIs, because they are _de facto_ stable) instead of actual DOIs in the GG XML is that this makes it a lot easier to access the actual PNGs for the TB treatment pages - the DOIs resolve to the deposition pages, not to the actual PNG.

**Comment by gsautter** [comment]
In a bid to make the change, I just found we're linking article -> figure with the "hasPart" relation, not with "cites" ... so where (what exact deposition) do you have this JSON snippet from?

Apart from that, using the aforementioned "hasPart" relation, we're linking article -> figure with all three of (1) the figure DOI (resolving to the deposition page), (2) a Plazi LSID starting with `urn:lsid:plazi.org:fig:` and ending in the page ID and bounding box within the article proper, and (3) the HTTP URI of the PNG that you mentioned.
In this system, (1) is the canonical referencing mechanism, (2) helps track changes to the image bounds in the IMF on the Plazi end (the deposition ID is an attribute to the figure caption), and (3) helps metadata consumers (e.g. early Ocellus) to get access to the actual PNG without having to retrieve the figure deposition metadata, simply saving a ton of requests.

**Comment by tcatapano** [comment]
@gsautter @myrmoteras  It woudl be `cito:cites`, but that would be for use in custom metadata and Zenodo would have to make the CITO chema available for use in custom metadata. That said, I do not see much downside to using the hasPart relation. It does seem safe to say that a figure *is* part of the article which contains it.

**Comment by gsautter** [comment]
I'd still be interested where this "cites" relationship comes from in the first place ... neither https://zenodo.org/record/1458424 (the figure) nor https://zenodo.org/record/1458416 (the parent article) shows anything to this avail. Could it be this one is from a manually created test treatment?

**Comment by tcatapano** [comment]
@gsautter, at first I was going to say that it should be in custom metadadata as `cito:cites`  following the guidance/example at:

https://codimd.web.cern.ch/k2hhtG5cR_qLncq14ouVgA#Data-model3

> You can use one of the above custom fields by prefixing them with their schema, using the following syntax: `"<vocabulary>:<term>": ["<value>"].  `

BUT then I noticed in the same document that there are examples which use "cites" as the predicate in relation statements, see: 

```
 "related_identifiers": [
          {"relation": "isPartOf", "identifier": "https://doi.org/10.11646/zootaxa.4459.1.5"},
          {"relation": "cites", "identifier": "https://zenodo.org/record/1458418/files/figure.png"},
          {"relation": "cites", "identifier": "https://zenodo.org/record/1458424/files/figure.png"}
```

The example also uses the png URLs, which I wouldnt recommend for article relations to figures, but aside from that, I dont know if the use of "cites" is a mistake, anticipating a feature yet to be released, or an actually implemented feature. At this point we should ask Alex.

**Comment by tcatapano** [comment]
I realize this is not the main point being made by @gsautter, but to the extent that it is an issue,  I feel that that the proper relationship between an article and a figure is "hasPart". For Treatments, it should be "cites" since the figures referred to from a treatment might well not be *part* of the treatment and may be referred to by multiple treatments.
